docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
+| [Dex](https://github.com/PatMartin/Dex)                                | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for reading, transforming, and visualizing data <br/> - Written in Groovy and Java on top of [JavaFX](https://openjfx.io/)                                                                                                                                                                                                                                                                          |
+| [CoreNLP](https://stanfordnlp.github.io/CoreNLP/)                      | Data cleaning and processing                                                                      | - Natural language processing toolkit <br/> - Linguistic annotations for text, such as sentiment and quote attributions <br/> - Support for eight languages                                                                                                                                                                                                                                                 |
+| [Smile](https://github.com/haifengl/smile)                             | - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building  | - Ready-made algorithms for machine learning and natural language processing <br/> - Linear algebra, graph, interpolation, and visualization tools <br/> - Smile provides a functional [Kotlin API](https://github.com/haifengl/smile/tree/master/kotlin), [Scala API](https://github.com/haifengl/smile/tree/master/scala), [Clojure API](https://github.com/haifengl/smile/tree/master/clojure), and more |
+| [Smile-NLP-kt](https://github.com/londogard/smile-nlp-kt)              | Data cleaning and processing                                                                      | - Kotlin rewrite of the [Scala](https://www.scala-lang.org/api/current/) implicits for the natural language processing part of Smile <br/> - Operations in the format of Kotlin extension functions and interfaces <br/> - Sentence breaking, stemming, bag of words, and other tasks                                                                                                                       |
+| [Weka](https://www.cs.waikato.ac.nz/ml/index.html)                     | - Data cleaning and processing <br/> - Data exploration and visualization  <br/> - Model building | - Collection of machine learning algorithms for data mining tasks   <br/> - Tools for data preparation, classification, regression, clustering, association rules mining, and visualization                                                                                                                                                                                                                 |
```

Comment:
Note: The last version of this library was released in 2021

Reply:
Let's keep